app/src/ui/branches/pull-request-badge.tsx

```diff
@@ -18,15 +18,10 @@ export class PullRequestBadge extends React.Component<
   public render() {
     const status = this.props.status
 
-    if (!status || status.totalCount === 0) {
-      return null
-    }
-
     return (
       <div className="pr-badge">
         <span className="number">#{this.props.number}</span>
-
-        <CIStatus status={status} />
+        {status && status.totalCount ? <CIStatus status={status} /> : null}
```

Comment:
Change `status && status.totalCount` to `status != null && status.totalCount > 0`.

Reply:
👍, and that’ll let you use `status != null && status.totalCount > 0 && <CIStatus status={status} />` since both `status != null` and `status.totalCount > 0` evaluate to booleans, which don’t get rendered.

Reply:
@j-f1 I'm just going to leave it as what @iAmWillShepherd said, because I think we should keep the explicit `null` around rather than relying on some obscure React behaviour about rendering booleans.

Reply:
> some obscure React behaviour about rendering booleans

It’s a [recommended practice](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored) in their docs.

Reply:
@j-f1 I'm glad it's in their docs (I honestly haven't tried rendering a boolean yet) but passing `null` is an organic convention we use elsewhere in the codebase, especially with functions that return `JSX.Element | null`.